corehq/apps/data_cleaning/models.py

```diff
@@ -877,6 +877,31 @@ def filter_query(self, query):
 class BulkEditColumnManager(models.Manager):
     use_for_related_fields = True
 
+    def create_session_defaults(self, session):
+        default_properties = {
+            BulkEditSessionType.CASE: (
+                'name', 'owner_name', 'date_opened', 'opened_by_username',
+                'last_modified', '@status',
+            ),
+        }.get(session.session_type)
+
+        if not default_properties:
+            raise NotImplementedError(f"{session.session_type} default columns not yet supported")
+
+        from corehq.apps.data_cleaning.utils.cases import (
+            get_system_property_label,
+            get_system_property_data_type,
+        )
```

Comment:
Do we still need to do local imports here, or has this code solved this issue and these can be extracted outside in a separate PR?

Reply:
adding to list of followup items to look into separate pr